Hazelcast.Examples/Client/ClientStatisticsExample.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Hazelcast.Client;
+using Hazelcast.Config;
+
+namespace Hazelcast.Examples.Client
+{
+    internal class ClientStatisticsExample
+    {
+        private static void Run(string[] args)
+        {
+            //This will enable client statistics 
+            Environment.SetEnvironmentVariable("hazelcast.client.statistics.enabled", "true");
+            
+            //set the statistics send period, default value is 3 seconds
+            Environment.SetEnvironmentVariable("hazelcast.client.statistics.period.seconds", "3");
+            
+            var config = new ClientConfig();
+            config.GetNetworkConfig().AddAddress("127.0.0.1");
+            var client = HazelcastClient.NewHazelcastClient(config);
+            //After client connected you can use Management Center to visualize client statistics
+            
```

Comment:
you may put some sleep here to give the user some time to visualise the statistics.

Reply:
Let me put some client activity instead